shoov/modules/custom/shoov_ci_build/shoov_ci_build.module

```diff
@@ -58,6 +69,97 @@ function  shoov_ci_build_trigger_ci_build_item($message, $ajax = 'nojs') {
 }
 
 /**
+ * Check permissions to un/subscribe account to the CI Build.
+ *
+ * @param $ci_build
+ *  The CI Build node object
+ * @param $account
+ *  The user object.
+ * @param $token
+ *  The access token
+ *
+ * @return bool
+ *  Returns TRUE if user has permissions to un/subscribe account to the CI Build.
+ */
+function shoov_ci_build_subscribe_user_access($ci_build, $account, $token) {
+  if (!user_is_logged_in()) {
```

Comment:
Do you mean `$account` is the account the should be un/subscribed or the `global $user...`?